webhook_to_fedora_messaging/main.py

```diff
@@ -35,13 +31,11 @@ def create_app():
 
     # Then load the variables up from the custom configuration file
     try:
-       confdata = get_config()
+        confdata = get_config()
+        main.config.from_mapping(confdata)
+        db.init_app(main)
```

Comment:
I don't think this should be run inside the try/except block, or could this call raise `ConfigError`?